### ISSUE TITLE
clear not found on RH8.1

### ISSUE THREAD

**Issue by marafa-sugarcrm** [lensapp/lens#1046]
**Describe the bug**
when attempting to exec into a pod, i see the terminal open up with  `exec kubectl exec -i -t -n kube-system calico-node-qcmmg -c calico-node "--" sh -c "clear; (bash || ash || sh)"` and then fail with `sh: clear: command not found`

**To Reproduce**
1. Go to pod view
2. Click on the terminal icon
3. Scroll down to the terminal box
4. See error `sh: clear: command not found`

**Expected behavior**
that the exec into the pod will work

**Screenshots**

**Environment (please complete the following information):**
- Lens Version: 3.6.6
- OS: OSX latest
- Installation method (e.g. snap or AppImage in Linux): brew
 
**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
Your logs go here...
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
your kubeconfig here
```

**Additional context**

**Comment by marafa-sugarcrm** [comment]
actually, it turned out the calico pod did not have clear installed.

**Comment by Nokel81** [comment]
Yes this has to do with the pod not having `clear` installed. A similar situation occurs if `bash` or `ash` is also not installed.

Given that `clear` is an executable and not a shell builtin. We probably should do something like what is listed here: https://askubuntu.com/questions/25077/how-to-really-clear-the-terminal
